webpack.config.js

```diff
@@ -130,7 +127,7 @@ module.exports = (env) => {
         IODIDE_JS_PATH: JSON.stringify(APP_PATH_STRING),
         IODIDE_CSS_PATH: JSON.stringify(CSS_PATH_STRING),
         IODIDE_BUILD_MODE: JSON.stringify((env && env.startsWith('dev')) ? 'dev' : 'production'),
-        IODIDE_BUILD_TYPE: JSON.stringify((env && env.includes('client-only')) ? 'standalone' : 'server'),
+        IODIDE_BUILD_TYPE: 'server',
```

Comment:
Due to recent PRs we've removed any use of `IODIDE_BUILD_TYPE` from the client code. Is `IODIDE_BUILD_TYPE` needed anywhere else at this point, specifically Travis?

Reply:
I don't think so.  I'll rebase this, which should retrigger a travis run to test it.  But we basically wouldn't be using travis to deploy at all anymore with this change (just the heroku deployment which is totally separate), so if the tests pass, I think we're good to go.